backends/dpdk/dpdkProgram.cpp

```diff
@@ -136,8 +136,10 @@ const IR::DpdkAsmProgram *ConvertToDpdkProgram::create(IR::P4Program *prog) {
     for (auto kv : structure->pipelines) {
         if (kv.first == "Ingress")
             kv.second->apply(*ingress_converter);
-        else if (kv.first == "Egress")
+        else if (kv.first == "Egress") {
```

Comment:
Please see if anything is required for EgressParser as well. We get instructions from parser as well.